src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -22,6 +22,7 @@ public sealed partial class ProcessStartInfo
         private string _directory;
         private string _userName;
         private string _verb;
+        private IReadOnlyCollection<string> _argumentList = Array.Empty<string>();
```

Comment:
While this may become moot if we move away from the constructor model, as it stands right now, this field should be marked `readonly`.